automation/taskcluster/lib/tasks.py

```diff
@@ -483,6 +483,7 @@ def get_architecture_and_build_type_from_variant(variant):
         )
 
     build_type = variant[len(architecture):]
+    build_type = build_type[0].lower() + build_type[1:]
```

Comment:
Nit: use `lower_case_first_letter()` in `automation/taskcluster/lib/util.py`

Reply:
Aha, that exists in `fenix`, not just `reference-browser`. I should've checked, thanks!